### ISSUE TITLE
how to use ssl

### ISSUE THREAD

**Issue by mk148a** [valiant1x/intense-pool#7]
hi, i have https pool web site, how to use this api on ssl port. I want to use 8119, i am try but its not opened but http api port 8117 is opened

**Comment by mk148a** [comment]
node is only open 8117 port, mean its http port its not opened 8119 ssl port

**Comment by cryptogrot** [comment]
@mk148a You could use Apache or Nginx in front as a proxy.

**Comment by mk148a** [comment]
@cryptogrot Thanks for your solution! But i dont understand how to use fron as a proxy? I use apache2 and Let's Encrypt ssl, everythinks seems ok but its not listen on ssl port.

**Comment by mk148a** [comment]
issue is still continue

**Comment by cryptogrot** [comment]
Did you see my answer: https://github.com/valiant1x/intense-pool/issues/9#issuecomment-359090884 ??

**Comment by mk148a** [comment]
@cryptogrot  yeah but my ngnix is give me error with this config file, its not support http2

**Comment by cryptogrot** [comment]
@mk148a So you will have to update your Nginx version.

**Comment by mk148a** [comment]
@cryptogrot its already upgraded version i have ubuntu 14.04 can you help me on slack or skype?
I need solution :S
